packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/WorkspaceSettings.tsx

```diff
@@ -468,20 +459,32 @@ export const WorkspaceSettings = () => {
         )}
       </Stack>
 
+      {isTeamAdmin && numberOfEditorsIsOverTheLimit && (
+        <MessageStripe justify="space-between">
+          You are no longer in a PRO team. Free teams are limited to 5 editor
+          seats. Some permissions might have changed.
+        </MessageStripe>
+      )}
+
+      {isTeamAdmin && hasMaxNumberOfEditors && (
+        <MessageStripe justify="space-between">
+          You&apos;ve reached the maximum amount of free editor seats. Upgrade
+          for more.
+          <MessageStripe.Action>Upgrade now</MessageStripe.Action>
+        </MessageStripe>
+      )}
+
       <div>
         <MemberList
           getPermission={user => getAuthorization(user, team)}
           getPermissionOptions={user => {
-            const yourAuthorization = activeWorkspaceAuthorization;
-
             // if changing the role will lead to extra seats, we want to
             // confirm any payment changes if required
             const confirmChange =
               confirmMemberRoleChange &&
               getAuthorization(user, team) === TeamMemberAuthorization.Read;
 
-            return yourAuthorization === TeamMemberAuthorization.Admin &&
-              user.id !== stateUser.id
+            return isTeamAdmin && user.id !== stateUser.id
```

Comment:
I've seen `stateUser` coming up multiple times. Is it the same as something like `currentUser`?

Reply:
Renamed